From 59818146b5f774ccc47b6e993f4c1e1de79a1df7 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 12 Dec 2024 14:03:01 +0800
Subject: [PATCH 284/301] AOSCOS: MIPS: loongson3-memcpy: use PTR_WD to fix
 build

Per commit fa62f39dc7e2 ("MIPS: Fix build error due to PTR used in more
places"), `PTR' was renamed to `PTR_WD' to avoid definition conflicts.

Follow this commit to fix build.

Fixes: "AOSCOS: memcpy optimization for loongson-3"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/loongson64/loongson3-memcpy.S | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/mips/loongson64/loongson3-memcpy.S b/arch/mips/loongson64/loongson3-memcpy.S
index 28b785363351..4c0b6b8474f0 100644
--- a/arch/mips/loongson64/loongson3-memcpy.S
+++ b/arch/mips/loongson64/loongson3-memcpy.S
@@ -37,13 +37,13 @@
 #define EXC(inst_reg,addr,handler)		\
 9:	inst_reg, addr;				\
 	.section __ex_table,"a";		\
-	PTR	9b, handler;			\
+	PTR_WD	9b, handler;			\
 	.previous
 
 #define EXCQ(inst,reg1,reg2,addr,handler)	\
 9:	inst	reg1, reg2, addr;		\
 	.section __ex_table,"a";		\
-	PTR	9b, handler;			\
+	PTR_WD	9b, handler;			\
 	.previous
 
 #endif
-- 
2.48.1

